cvat/apps/engine/permissions.py

```diff
@@ -442,17 +465,35 @@ def create(cls, request: ExtendedRequest, view: ViewSet, obj: Task | None, iam_c
             assignee_id = request.data.get('assignee_id') or request.data.get('assignee')
             owner = request.data.get('owner_id') or request.data.get('owner')
 
-            for scope in cls.get_scopes(request, view, obj):
+            scopes = cls.get_scopes(request, view, obj)
+
+            if cls.Scopes.UPDATE_ORGANIZATION in scopes:
+                # consider this case as deleting a task in the org A and creating a new one in the org B
+                if obj.project is not None:
```

Comment:
```suggestion
                if obj.project_id:
```
Accessing `project` can lead to fetching from the DB, if the project is not prefetched.